nebula/src/nebula/video_storage/storage.py

```
@@ -0,0 +1,297 @@
+                check=True,
+                timeout=3600,  # 1 hour timeout
+            )
+
+            logger.info("HLS conversion successful: %s", playlist_path)
+
+            # Get video duration
+            duration = self._get_video_duration(video_path)
+            return duration
+
+        except subprocess.CalledProcessError as e:
+            logger.error("FFmpeg error: %s", e.stderr)
```

Ruff (0.14.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)